.grit/patterns/terraform_cloudflare_v5.grit

```diff
@@ -214,8 +214,21 @@ pattern terraform_cloudflare_v5() {
   inline_cloudflare_block_to_list(`minify`) as $block where { $block <: within `resource "cloudflare_zone_settings_override" $_ { $_ }` },
   inline_cloudflare_block_to_list(`mobile_redirect`) as $block where { $block <: within `resource "cloudflare_zone_settings_override" $_ { $_ }` },
   inline_cloudflare_block_to_list(`nel`) as $block where { $block <: within `resource "cloudflare_zone_settings_override" $_ { $_ }` },
-  inline_cloudflare_block_to_list(`security_header`) as $block where { $block <: within `resource "cloudflare_zone_settings_override" $_ { $_ }` }    
+  inline_cloudflare_block_to_list(`security_header`) as $block where { $block <: within `resource "cloudflare_zone_settings_override" $_ { $_ }` },
+
+  // provider
```

Comment:
Note the workflow will overwrite these. I would put any manual changes in a different file/pattern that composes this pattern.

Reply:
will do! thanks.